source/precharge/bms/orion.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _ORION_H_
+#define _ORION_H_
+
+#include "can_parse.h"
+
+#include <stdbool.h>
+
+bool check_errors();
```

Comment:
Typically for public functions try to make their name also reference which module they are from. Check_errors is a little generic. Also, code standards for make function names camel case (viewable in confluence for more info)